easybuild/easyconfigs/m/multicharge/multicharge-0.2.0-gfbf-2022a.eb

```diff
@@ -0,0 +1,39 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeNinja'
+
+name = 'multicharge'
+version = '0.2.0'
+
+homepage = 'https://github.com/grimme-lab/multicharge'
+description = """Electronegativity equilibration model for atomic partial charges."""
+
+toolchain = {'name': 'gfbf', 'version': '2022a'}
+
+github_account = 'grimme-lab'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['953e2ace2f4035b1fa8ecf680f90b5ce6ad5caae17c8d8ccbc2578b92b69d3e7']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Ninja', '1.10.2'),
+    ('binutils', '2.38'),
```

Comment:
This does not need binutils as builddep, only things in GCCcore needs that. Please remove

Reply:
Done. Thanks for finding this!